corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -190,61 +191,97 @@ <h4 class="panel-title panel-title-nolink">{% trans "Default Search Properties"
+                  {% trans "Copy case list filter" %}
+                </button>
+                {% trans "Example: age > 5" %}
+              </p>
+            </div>
+          </div>
+          <div class="form-group">
+            <label for="claim-relevant-condition" class="control-label {% css_label_class %}">
+              {% trans "Search Condition" %}
+              <span class="hq-help-template" data-title="{% trans "Search Condition" %}"
+                    data-content="{% trans_html_attr "If this expression evaluates to false, the search will not be performed." %}">
+              </span>
+            </label>
+            <div class="{% css_field_class %}">
+              <textarea data-bind="value: relevant"
+                        class="form-control vertical-resize"
+                        id="claim-relevant-condition"
+                        spellcheck="false"
+              ></textarea>
+            </div>
+          </div>
+          <div class="form-group">
+            <label class="control-label {% css_label_class %}" for="search-default-relevant">
               {% trans "Don't claim cases already owned by the user" %}
             </label>
+            <div class="checkbox {% css_field_class %}">
+              <label>
+                <input type="checkbox" id="search-default-relevant" data-bind="checked: default_relevant">
+              </label>
+            </div>
           </div>
-        </div>
-        <div class="form-group">
-          <label for="blacklisted-user-ids" >
-            {% trans "Don't search cases owned by the following ids" %}
-          </label>
-          <input type="text"
-                 data-bind="value: blacklistedOwnerIdsExpression"
-                 class="form-control"
-                 id="blacklisted-user-ids"
-                 spellcheck="false"
-          />
-          <p class="help-block">
-            {% blocktrans %}
-              An XPath expression that will evaluate to a space separated list of ids. For example: instance('commcaresession')/session/context/userid or 'a1c0148dc2120c6b1762f5ac5aba2a15'
-            {% endblocktrans %}
-          </p>
-        </div>
-        <div class="form-group">
-          <div class="checkbox">
-            <label>
-              <input type="checkbox" data-bind="checked: includeClosed"> {% trans "Include closed cases in search results" %}
+          <div class="form-group">
+            <label for="blacklisted-user-ids" class="control-label {% css_label_class %}">
+              {% trans "Don't search cases owned by the following ids" %}
+              <span class="hq-help-template" data-title="{% trans "Ignore Owners" %}"
+                    data-content="{% trans_html_attr "An XPath expression that will evaluate to a space separated list of ids." %}">
+              </span>
             </label>
+            <div class="{% css_field_class %}">
+              <textarea data-bind="value: blacklistedOwnerIdsExpression"
```

Comment:
In theory, we could now remove all these other things since you could add them in your new filter box, but I guess it is nicer to have them as UI components.

Reply:
Yeah, I'm not really keen on doing a migration, especially when it'll make for a less-friendly user experience. Although...it might be nice to get rid of the default search properties panel, just because it takes up a good deal of space on a page that's already really long (case list settings). Anyway, I'll take a quick look at what settings projects are currently using in production.

Reply:
Noting this isn't worth looking into anymore now that the filter is on results. Migrating existing search filters to instead filter results would degrade performance for current users.